vgscollect/src/main/java/com/verygoodsecurity/vgscollect/core/model/network/VGSError.kt

```diff
@@ -30,6 +30,9 @@ enum class VGSError(val code:Int, val messageResId:Int) {
     ),
     FILE_SIZE_OVER_LIMIT(1103,
         R.string.error_file_size_validation
```

Comment:
Change the error message to `error_file_size_validation` to avoid specifying a size.